_shared/project/tests/pytest_plugins/db.py

```diff
@@ -0,0 +1,22 @@
+from os import environ
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from {{ cookiecutter.package_name }}.db import Base
+
+
+@pytest.fixture(scope="session")
+def db_engine():  # pragma: no cover
+    engine = create_engine(environ["DATABASE_URL"])
+
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
+
+    return engine
+
+
+@pytest.fixture(scope="session")
+def db_sessionfactory():  # pragma: no cover
+    return sessionmaker()
```

Comment:
Again, projects can override these fixtures in their `conftest.py`'s or add this entire file to their ignore lists.